core/src/i18n/en.yaml

```diff
@@ -580,7 +580,7 @@ components:
   error-panel:
     error: An error occured. The component cannot be rendered.
   file-input:
-    drag-file: Drag file here
+    drag-file: Drag your file here or click to browse your computer
```

Comment:
Maybe something like: `Choose a file or drag it here` or `Drag your file here or click to upload`